lib/chat_api/emails.ex

```diff
@@ -7,11 +7,12 @@ defmodule ChatApi.Emails do
   alias ChatApi.Emails.Email
   alias ChatApi.Users.{User, UserSettings}
 
-  def send_new_message_alerts(message, account_id, conversation_id) do
-    account_id
+  def send_new_message_alerts(message_obj) do
```

Comment:
small nitpick -- instead of `message_obj` can we just call this `message` and maybe at a typespec above the method to make it clear that this is a Message struct and not just a string, e.g.
```elixir
alias ChatApi.Messages.Message

# ...

@spec send_new_message_alerts(Message.t()) :: [any]
def send_new_message_alerts(message) do
  # ...
end
```

this is something we haven't done a great job at so far, but I'd like to start adding more typespecs moving forward -- it would a great help if you could add some to the methods you updated 😄

Reply:
Sure thing.